conda/recipes/rmm/recipe.yaml

```diff
@@ -20,8 +20,23 @@ source:
 build:
   string: cuda${{ cuda_major }}_py${{ py_buildstring }}_${{ date_string }}_${{ head_rev }}
   script:
-    content:
-      - ./build.sh -v clean rmm --cmake-args=\"-DCMAKE_INSTALL_LIBDIR=lib\"
+    content: |
+      # Remove `-fdebug-prefix-map` line from CFLAGS and CXXFLAGS so the
+      # incrementing version number in the compile line doesn't break the
+      # cache
+      set -x
+      export CFLAGS=$(echo $CFLAGS | sed -E 's@\-fdebug\-prefix\-map[^ ]*@@g')
+      export CXXFLAGS=$(echo $CXXFLAGS | sed -E 's@\-fdebug\-prefix\-map[^ ]*@@g')
```

Comment:
We should also export this in CUDAFLAGS I expect.

Reply:
Agreed. Tried to capture a better recommendation here: https://github.com/rapidsai/rapids-cmake/issues/798#issuecomment-2760017833

Please feel free to suggest further improvements